docs/modules/ROOT/pages/list-of-metrics.adoc

```diff
@@ -94,18 +94,24 @@ in milliseconds).
 
 |status
 |Information regarding the job's status. Potential values:
-0 - submitted;
-1 - initialization phase;
-2 - running;
-3 - suspended;
-4 - exporting snapshot;
-5 - completing phase;
-6 - failed;
-7 - completed successfully;
+
+* 0 - submitted
+* 1 - initialization phase
+* 2 - running
+* 3 - suspended
+* 4 - exporting snapshot
+* 5 - completing phase
+* 6 - failed
+* 7 - completed successfully
 
 |userCancelled
-|Details on whether the user canceled the job. Potential values: 1 - canceled by the user, 0 - otherwise.
+|Details on whether the user canceled the job. Potential values:
+
+* 1 - canceled by the user
+* 0 - otherwise
+
 Exposed via xref:maintain-cluster:monitoring.adoc#via-job-api[Job API] only.
+Note that this metric can be retrieved only if you enable the `storeMetricsAfterJobCompletion` setting for your jobs. See xref:pipelines:configuring-jobs.adoc#storing-metrics[Storing Job Metrics] to learn how to enable it.
```

Comment:
```suggestion
To retrieve this metric, you must enable `storeMetricsAfterJobCompletion` for your jobs. For further information on enabling this behavior, see xref:pipelines:configuring-jobs.adoc#storing-metrics[Storing Job Metrics].
```

Reply:
Not applicable anymore @rebekah-lawrence ; I got the internals wrong.